src/tracer.js

```diff
@@ -96,7 +100,16 @@ export default class Tracer {
     this.registerInjector(opentracing.FORMAT_BINARY, binaryCodec);
     this.registerExtractor(opentracing.FORMAT_BINARY, binaryCodec);
 
-    this._reporter.setProcess(this._serviceName, Utils.convertObjectToTags(this._tags));
+    const uuid = Utils.getRandom64().toString('hex');
+    this._tags[constants.TRACER_CLIENT_ID_TAG_KEY] = uuid;
+    this._process = {
+      serviceName: serviceName,
+      tags: Utils.convertObjectToTags(this._tags),
+      uuid: uuid,
```

Comment:
Why is this not named processId as it's not a UUID its just a random number?

Reply:
This is somewhat unfortunate. I could pull in another dependency (npm uuid) and actually generate a uuid but given the collision can only occur on one host, I didn't see the need to generate a real uuid.

bah, ill try to look for a very light weight uuid generator maybe this (https://gist.github.com/jed/982883) and inline it. I rather not pull in a new dependency.